vault/datadog_checks/vault/vault.py

```diff
@@ -43,13 +44,13 @@ def check(self, instance):
             api['check_leader'](config, tags)
             api['check_health'](config, tags)
         except ApiUnreachable:
-            return
+            raise
```

Comment:
At this point we have already set the service check to critical and logged the exception. Why raise here instead of returning?

Reply:
To let the agent know that the check was not successful. And see the error in `datadog-agent status` or in the web-ui.